src/main/java/bit/javaoop/FixedBudgetManager.java

```diff
@@ -2,44 +2,43 @@
 
 public class FixedBudgetManager extends Manager {
 
+    private static final int SATISFYING_SALARY = 20_000;
+    private static final int SPARE_BUDGET_LIMIT = 15_000;
     private double fixedBudget;
 
     public FixedBudgetManager(String name, double salary) {
         super(name, salary);
     }
 
     public boolean isSatisfied() {
-        if (this.getSalary() > 20_000 && (this.currentBudget() < 15_000)) {
-            return true;
-        }
-        return false;
+        return getSalary() > SATISFYING_SALARY && (currentBudget() < SPARE_BUDGET_LIMIT);
     }
 
     @Override
     public boolean hireNewEmployee(Employee employee) {
-        if ((this.currentStuffSalary() + employee.getSalary()) <= this.getFixedBudget()) {
-            this.getMyStaffList().add(employee);
+        if ((currentStuffSalary() + employee.getSalary()) <= getFixedBudget()) {
+            getMyStaffList().add(employee);
             return true;
         } else {
```

Comment:
właściwie jak mamy jednego ifa z returnem to nie trzeba dawać elsa można by po prostu za ifem dać:
```
System.out.println("You don't have enought money to hire new employee.");
return false;
```
i tak jak wejdzie w ifa to return nam wyskoczy z funkcji także wyjdzie na to samo.